crates/core/src/db/datastore/traits.rs

```diff
@@ -438,6 +438,24 @@ pub trait MutTxDatastore: TxDatastore + MutTx {
     fn drop_constraint_mut_tx(&self, tx: &mut Self::MutTx, constraint_id: ConstraintId) -> super::Result<()>;
     fn constraint_id_from_name(&self, tx: &Self::MutTx, constraint_name: &str) -> super::Result<Option<ConstraintId>>;
 
+    // Row-level security
+    fn create_row_level_security_mut_tx(
```

Comment:
Does this need to be part of the datastore trait?

Reply:
Yes, is required because the accessors elsewhere

Reply:
The datastore traits are a little bloated IMO.

We should really think of the database as being implemented using the datastore. In other words, the datastore should be suitably generic so as to allow multiple database implementations. RLS is not a feature of a datastore but rather a database. So I would argue these methods are better suited to RelationalDB than anything else.

Reply:
I that deserve another pr.